orchestra/static/orchestra/common/css/orchestra.scss

```diff
@@ -1,6 +1,7 @@
-@import 'orchestra/static/orchestra/common/css/registration.scss';
+@import 'orchestra/static/accounts/scss/registration';
```

Comment:
according to @derek-schultz the proper way is to only have `_filename.scss` and then the scss import works as i have it. we haven't followed this convention in orchestra but should going forward